src/main/java/com/desiato/puresynth/services/SessionService.java

```diff
@@ -6,22 +6,36 @@
 import com.desiato.puresynth.repositories.SessionRepository;
 import com.desiato.puresynth.repositories.UserRepository;
 import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
 import org.springframework.stereotype.Service;
 
 import java.util.Optional;
 import java.util.UUID;
 
 @AllArgsConstructor
+@Slf4j
 @Service
 public class SessionService {
 
     private final SessionRepository sessionRepository;
     private final UserRepository userRepository;
 
     public Session createSession(User user) {
+        if (user.getEmail() == null) {
+            throw new IllegalStateException("User email must not be null");
+        }
         String tokenValue = UUID.randomUUID().toString();
         Session newSession = new Session(tokenValue, user.getId());
-        sessionRepository.save(newSession);
+        log.info("Token to be assigned: {}", tokenValue);
```

Comment:
You can't log a token

Reply:
I know, but at that point, I needed to know exactly what was the token and if I was getting the token

Reply:
I absolutely should have removed it before pushing